packages/replayer/src/replay.utils.ts

```diff
@@ -105,6 +116,16 @@ export const createReplayPage: (options: {
   return page;
 };
 
+const installVirtualEventLoop = (page: Page) =>
+  page.evaluateOnNewDocument(`
+    const installVirtualEventLoop = window.__meticulous?.replayFunctions?.installVirtualEventLoop;
+    if (installVirtualEventLoop) {
+      installVirtualEventLoop();
+    } else {
+      console.error("Could not install virtual event loop since window.__meticulous.replayFunctions.installVirtualEventLoop was null or undefined");
```

Comment:
Maybe this should throw an error?

Reply:
Throwing an error wouldn't do anything other than log to the developers console, because this is evaluated in global context on document load.

Do we currently capture and log uncaught exceptions during replay?